.gitignore

```diff
@@ -61,3 +63,12 @@ test/onnx/models
 # VS2017 and VSCode config files.
 .vscode
 .vs
+
+# documentation artifacts
+build/
```

Comment:
Shouldn't this be covered by L58 already?

Reply:
Yes, thanks for catching that. Updated in [baa4e55](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/1727/commits/baa4e553c8ab628d0a0b07f052d529881b91f407)